src/Type/ContentItemRegionType.php

```diff
@@ -13,6 +12,7 @@
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\Form\AbstractType;
 use Symfony\Component\Translation\TranslatorInterface;
+use Symfony\Component\Validator\Constraints as Assert;
```

Comment:
Zou dit gewoon `Constraints` noemen, niet `Assert`.  Lijkt me alleen maar verwarrend.

Verder gebruik je allen de `NotBlank`, dus je kan ook alleen die hier importeren.

Reply:
Dit is hoe Symfony het zelf doet en ik heb mijzelf dit ook aangeleerd 🤷🏻‍♂️

Reply:
https://github.com/search?l=PHP&q=org%3Azicht+NotBlank&type=Code

Als je in de code van Zicht zoekt, dan is 'Assert' in de minderheid, op pagina 8 kom ik het voor het eerst pas tegen.  Dus binnen zicht is het blijkbaar niet zo gebruikelijk...

Dit is voor mij niet blocking, maar persoonlijk zou ik het doen zoals de meeste code van Zicht het nu lijkt te doen.  Your choice :+1:

Reply:
Ik heb het ook nog nooit zo gedaan eigenlijk, ik zet hier altijd direct `new NotBlank()` neer